umbraco-cloud/upgrades/version-specific-upgrades/migrate-from-umbraco-7-to-8.md

```diff
@@ -0,0 +1,161 @@
+
+Running the `echo > deploy` command will generate a new marker. Move forward with the migration based on the marker:
+
+* `deploy-failed`
+  * Something failed during the check
+  * Run `echo > deploy-clearsignatures` followed by `echo > deploy` to clear up the error
+* `deploy-complete`
+  * Everything checks out: Move on to the next step
+
+## Step 3: Setup custom code for Umbraco 8
+
+Umbraco 8 is different from Umbraco 7 in many ways. This means that in this step, all custom code, controllers, and models need to be reviewed and rewritten for Umbraco 8.
+
+### Example of changes that need to be made
+
+One of the changes made, is how published content is rendered through Template files. Due to this, it will be necessary to update **all** Template files (`.cshtml`) to reflect these changes.
+
+Read more about these changes in the [IPublishedContent section of the Documentation](https://docs.umbraco.com/umbraco-cms/reference/querying/ipublishedcontent).
+
+* Template files need to inherit from `Umbraco.Web.Mvc.UmbracoViewPage<ContentModels.HomePage>` instead of `Umbraco.Web.Mvc.UmbracoTemplatePage<ContentModels.HomePage>`.
+* Template files need to use `ContentModels = Umbraco.Web.PublishedModels` instead of `ContentModels = Umbraco.Web.PublishedContentModels`.
+* `@Model.Value("propertyAlias")` replaces `@Umbraco.Field("propertyAlias")`.
+* `@Model.PropertyAlias` replaces `@Model.Content.PropertyAlias`.
+
+Depending on the size of the project and the amount of custom code and implementations, this step is going to require a lot of work.
+
+## Step 4: Deploy and test on Umbraco Cloud
+
+Once the Umbraco 8 project runs without errors on the local setup, the next step is to deploy and test on the Cloud Development environment.
+
+1. Push the migration and changes to the Umbraco Cloud Development environment
+
+{% hint style="info" %}
+The deployment might take a bit longer than normal.
+
+To track the process, keep an eye on the deploy markers in `site/wwwroot/data` using KUDU.
+{% endhint %}
+
+2. Progress through the steps based on the deployment result:
+    * `deploy-failed`: Run `echo > deploy-clearsignatures` followed by `echo > deploy` to clear up the error.
+    * `deploy-complete`: The Development environment has been upgraded.
+3. Transfer Content and Media from the local clone to the Development environment.
+4. Test **everything** on the Development environment.
+5. Deploy to the Live environment.
+
+## Step 5: Going live
+
+Once the migration is complete, and the Live environment is running without errors, the site is almost ready for launch.
+
+1. Setup rewrites on the Umbraco 8 site.
+2. Assign hostnames to the project.
+    * Hostnames are unique, and can only be added to one Cloud project at a time.
+
+## Related information
+
+* [Content Migration for Umbraco CMS - 7 to 8](https://docs.umbraco.com/umbraco-cms/fundamentals/setup/upgrading/version-specific/migrate-content-to-umbraco-8.md)
```

Comment:
Link doesn't work (Broken Link)

Reply:
The article linked to, will be added with this PR. That's why the link doesn't work at the moment.

Reply:
Got it.. 💪